testing/cffi1/test_ffi_obj.py

```diff
@@ -32,7 +32,11 @@ def test_ffi_cache_type():
     assert ffi.typeof("int[][10]") is ffi.typeof("int[][10]")
     assert ffi.typeof("int(*)()") is ffi.typeof("int(*)()")
 
+@pytest.mark.thread_unsafe(reason="May not pass if other threads unexpectedly trigger the gc")
 def test_ffi_type_not_immortal():
+    if hasattr(sys, '_is_gil_enabled') and not sys._is_gil_enabled():
+        pytest.skip("fails on free-threaded builds")
+
```

Comment:
This is failing on the GIL-enabled build too - https://github.com/ngoldbaum/cffi-ft/actions/runs/16023540072/job/45205948721

I think this change is incorrect and something else is causing this to fail. @kumaraditya303 are looking at it. Again, sorry for the noise...